### PR TITLE
Installation instructions for developers.

### PR DESCRIPTION
**Pull request by lyzkov**

### CODE REVIEW

File: doc/INSTALLATION.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+At first clone repository from github:
+```
+git clone git@github.com:seventag/seventag.git
+```
+
+Then configure server and project the same way as usual Symfony project.
+More instructions are available in official symfony [documentation](http://symfony.com/doc/current/book/installation.html).  
+You can also check if your machine meets [minimal requirements](https://7tag.org/docs/software-requirements/).
+
+**Remember to set up permissions for `var/cache`, `var/logs` and `/var/spool`.**
```
Comment (kallosz) [inline doc/INSTALLATION.md]: also `web/containers`

File: doc/INSTALLATION.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Symfony vendors and frontend dependencies can be installed easily by ant task:
+```
+ant build
```
Comment (michalsikora) [inline doc/INSTALLATION.md]: it loads mocking data for tests, you should after this run `gulp build`
or maybe we should create new subtask in ant to easily resolve problems with instalations?

**Comment by michalsikora** [comment]
rebase needed

File: doc/INSTALLATION.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+Create database schema:
+```
+bin/console doctrine:schema:create
+```
+
+At the end load data fixtures in order to work on prepared data examples.
+```
+php bin/console doctrine:fixtures:load
```
Comment (michalsikora) [inline doc/INSTALLATION.md]: Before this command you need to load schema, now you are trying to load fixtures on empty database

**Pull request merged**